include/RAJA/forall.hxx

```diff
@@ -301,4 +301,39 @@ RAJA_INLINE void forall_Icount(const Index_type* idx,
 
 }  // closing brace for RAJA namespace
 
+
+/*!
```

Comment:
This is used from test-forall (TYPED_TEST_P(ForallTest, BasicForallIcount)).  Do we need to rethink this?